stuff/cron/aggregate_feedback.py

```diff
@@ -76,16 +97,25 @@ def fill_rank_cutoffs(dbconn):
         return [rank_cutoff(row[0], row[1]) for row in cur]
 
 
-def get_weighting_factor(score, rank_cutoffs):
-    '''Gets the user vote weighting factor based on user's
-    score stored in User_Rank table and according to the
-    User_Rank_Cutoffs scores and classnames'''
-    if score is None:
+def get_weighting_factor(score, rank_cutoffs, before_ac):
```

Comment:
será posible agregarle [tipos](https://docs.python.org/3/library/typing.html)?

```suggestion
  def get_weighting_factor(score: float, rank_cutoffs: ..., weighting_factors: Mapping[str, float]) -> float: